modules/service/src/main/scala/lucuma/odb/service/ObservingModeServices.scala

```diff
@@ -47,7 +47,7 @@ sealed trait ObservingModeServices[F[_]] {
 
 object ObservingModeServices {
 
-  def instantiate[F[_]: MonadCancelThrow](using Services[F]): ObservingModeServices[F] =
+  def instantiate[F[_]: {MonadCancelThrow, Services}]: ObservingModeServices[F] =
```

Comment:
That syntax is new to me.  Is it the same as `F[_]: MonadCancelThrow: Services` ?

Reply:
Yes, it came on 3.6 with SIP-64
https://docs.scala-lang.org/sips/sips/typeclasses-syntax.html#2-new-syntax-for-aggregate-context-bounds